go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/bufbuild/connect-grpcreflect-go
 
-go 1.18
```

Comment:
This was needed to use `atomic.Bool`, which is quite convenient.

Reply:
Yep, makes sense - we should be good to bump our minimum supported version to 1.19.